src/main/webapp/api.js

```diff
@@ -0,0 +1,16 @@
+// Create the script tag, set the appropriate attributes
+var script = document.createElement('script');
+
+// Edit key between 'key=' and '&locat...' to our new API key once you pull this file.
+// Then add this file to .gitignore
+script.src = 'https://maps.googleapis.com/maps/api/js?key=AIzaSyB-kjcAxxbrQeJJEvSg5XgkTmGI4aZLS5E&libraries=places';
```

Comment:
Just so Fernando/Mengyi know, this is an old API key that has already been deleted